.github/workflows/ci_test.yml

```diff
@@ -52,3 +52,22 @@ jobs:
             echo export BAZEL_PATH=/d/a/addons/addons/bazel-%BAZEL_VERSION%-windows-x86_64.exe  >>D:\a\_temp\msys\msys64\home\runneradmin\.bash_profile
             echo export PATH=/c/hostedtoolcache/windows/Python/${{ matrix.python-version }}/x64/:$PATH  >>D:\a\_temp\msys\msys64\home\runneradmin\.bash_profile
             msys2do ./tools/ci_testing/addons_cpu.sh
+  sanity-cpu:
+    name: Sanity Check
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ['3.7.5']
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: CI sanity check py${{ matrix.python-version }}
+        run: |
+          set -x -e
+          python -m pip install wheel setuptools
+          wget --quiet -nc https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+          chmod +x bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+          ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh --user
```

Comment:
Please remove L69-72 as they're not needed. After that looks good to merge. Thanks!